lib/Graph/Graph.cpp

```diff
@@ -2993,3 +2993,19 @@ bool Function::verify() const {
   }
   return isValid;
 }
+
+SaveNode *glow::getOutputSave(Function* F, Placeholder *PH) {
+  // if parent is set for PH, check if it is the same as provided Function.
+  auto* PHP = PH->getParent();
```

Comment:
Clang format please. You can run the script 

./utils/format.h  or git-clang-format on the staged changes.